lib/util.js

```diff
@@ -0,0 +1,49 @@
+const { Op } = require("sequelize");
+const db = require("../sequelize/db");
+const pg = require("../database/postgres/pg");
+const es = require("../database/elasticsearch/elastic");
+const logger = require("./log")(__filename);
+
+const util = {};
+
+util.cleanAnonymous = async () => {
+  logger.info("Checking expired anonymous accounts...");
+  try {
+    const { Accounts } = db.getModels();
+    const userAccount = await Accounts.findAll({
+      attributes: ["id", "username"],
+      where: {
+        [Op.and]: [
+          {
+            createdAt: {
+              [Op.lt]: new Date(new Date() - 5 * 24 * 60 * 60 * 1000),
+            },
+          },
+          { email: null },
+        ],
+      },
+    });
+    logger.info(`${userAccount.length} expired accounts are found`);
+    return Promise.all(
+      userAccount.map(async (e) => {
+        const pgDbExists = await pg.userHasPgAccount(e.username);
```

Comment:
Will lines 29-32 be guaranteed to be called in sequential order?

Reply:
Since the callback is `async`, the `await`s in front of each functions will guarantee that those functions run sequentially in my opinion.